README.md

```
@@ -39,3 +40,30 @@
 # Remove services' containers only
 pnpm dev:down
 ```
+
+#### Docker containers:
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)